scalikejdbc-streams/src/test/resources/jdbc.properties

```diff
@@ -0,0 +1,4 @@
+driverClassName=org.h2.Driver
+url=jdbc:h2:mem:scalikejdbc;LOG=0;CACHE_SIZE=65536;LOCK_MODE=0;UNDO_LOG=0
+user=
+password=
```

Comment:
Is there any reasons can't use [`scalikejdbc-core/src/test/resources/*.properties`](https://github.com/scalikejdbc/scalikejdbc/tree/636a53ea99b6764e8fb907b216b63e6db7066186/scalikejdbc-core/src/test/resources) ?

We should copy to `scalikejdbc-streams/src/test/resources/jdbc_$SCALIKEJDBC_DATABASE.properties` from `scalikejdbc-streams/src/test/resources/jdbc.properties` in 
[travis_before.sh](https://github.com/scalikejdbc/scalikejdbc/blob/636a53ea99b6764e8fb907b216b63e6db7066186/travis_before.sh#L5-L10) if we can't use [`scalikejdbc-core/src/test/resources/*.properties`](https://github.com/scalikejdbc/scalikejdbc/tree/636a53ea99b6764e8fb907b216b63e6db7066186/scalikejdbc-core/src/test/resources)